src/api/todoApi.js

```diff
@@ -9,3 +9,12 @@ export const getTodos = async () => {
     console.log(e);
   }
 };
+
+export const addTodoToDB = async (item) => {
+  try {
+    let response = await axios.post(config.API_URL, item);
+    return response.data;
+  } catch (e) {
+    console.log(e);
```

Comment:
If we got an error in this case function will return undefined and you will put it in state. Lets catch this on up level and if we got an error show to user exception